pkgs/development/rocm-modules/5/default.nix

```diff
@@ -10,11 +10,15 @@
 , ffmpeg_4
 , libjpeg_turbo
 , rapidjson-unstable
+, default-gcc-version
 }:
 
 let
   rocmUpdateScript = callPackage ./update.nix { };
 in rec {
+
+  __attrsFailEvaluation = default-gcc-version == 13;
```

Comment:
Should we still be doing this if we're including bee20868efd25a3edce1f97d7d0fc68969864195 and c1c5fa671f584daaf7690e465b04af48edafc7fb?
AFAIK this will still fail to evaluate since the default gcc version will be 13 regardless of the package-specific stdenv changes.

Reply:
Ah wow thank you for catching that!

Reply:
Fixed in latest force-push to fa043b7b15ec